net.agalwood.Motrix.yml

```diff
@@ -0,0 +1,160 @@
+app-id: net.agalwood.Motrix
+branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+# Electron doesn't use a traditional locale format so separate-locales is useless.
+separate-locales: false
+command: start-motrix
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  # Copied from shared-modules/python2.7/python-2.7.json since it's only used at build time
+  # - name: python-2.7
+  #   sources:
+  #     - type: archive
+  #       url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+  #       sha256: b62c0e7937551d0cc02b8fd5cb0f544f9405bafc9a54d3808ed4594812edef43
+  #   config-opts:
+  #     - --enable-shared
+  #     - --with-ensurepip=yes
+  #     - --with-system-expat
+  #     - --with-system-ffi
+  #     - --enable-loadable-sqlite-extensions
+  #     - --with-dbmliborder=gdbm
+  #     - --enable-unicode=ucs4
+  #   post-install:
+  #     - chmod 644 $FLATPAK_DEST/lib/libpython2.7.so.1.0
+  #   cleanup:
+  #     - '*'
+
+  - name: yarn
+    buildsystem: simple
+    build-commands:
+      - cp -a * $FLATPAK_DEST
+    # Only used for building, so clean it up afterwards.
+    cleanup:
+      - "*"
+    sources:
+      - type: archive
+        url: https://github.com/yarnpkg/yarn/releases/download/v1.22.10/yarn-v1.22.10.tar.gz
+        sha256: 7e433d4a77e2c79e6a7ae4866782608a8e8bcad3ec6783580577c59538381a6e
+        x-checker-data:
+          type: anitya
+          project-id: 13363
+          url-template: https://github.com/yarnpkg/yarn/releases/download/$version/yarn-v$version.tar.gz
+  - name: motrix
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory.
+      append-path: /usr/lib/sdk/node14/bin:/app/yarn/bin:/run/build/motrix/flatpak-node/chromedrive
+      env:
+        # Don't add ELECTRON_CACHE
+        XDG_CACHE_HOME: /run/build/motrix/flatpak-node/cache
+        # npm_config_nodedir: /usr/lib/sdk/node14
+        npm_config_offline: "true"
+        npm_config_no_save: "true"
+        npm_config_cache: /run/build/motrix/flatpak-node/npm-cache
+        npm_config_loglevel: verbose
+        # https://stackoverflow.com/questions/25146976/can-i-get-npm-gyp-to-use-ccache
+        CXX: ccache g++
+        CC: ccache gcc
+      arch:
+        # https://github.com/sass/node-sass/issues/3033#issuecomment-763180778
+        # The architecture of Electron, see https://electronjs.org/docs/tutorial/support#supported-platforms
+        # for supported architectures.
+        i386:
+          env:
+            npm_config_arch: ia32
+            npm_config_target_arch: ia32
```

Comment:
neither `i386` nor `arm` are supported anymore by the runtime so it's unnecessary.

(it's harmless though, just clutter)

Reply:
Well, let's just leave it there? As they are supported by electron and someone might find it useful